### PR TITLE
Added documentation for new geo-triggering webhook event

### PR DESCRIPTION
**Pull request by toroduque**

### CODE REVIEW

File: docs/Webhooks/Geo-triggering.md [hunk 1]
```diff
@@ -1,183 +1,207 @@
...
+### `appInfo` fields
 
-If you’d like to Create a Destination using [Config API](../APIs/Config%20API/Overview.md), you’ll use the following endpoints:
+This table details the fields within the `appInfo` object, providing information about the application.
 
+| Field                                     | Description                                                                 |
+|-------------------------------------------|-----------------------------------------------------------------------------|
+| `appInfo.appBuildVersion`                 | Build version of the application.                                           |
+| `appInfo.customEventMetaData`             | Custom metadata related to the event.                                        |
+| `appInfo.customEventMetaData.[KEY]` | The key of the custom event data key/value pair.                                               |
```
Comment (andrew-stirling) [inline docs/Webhooks/Geo-triggering.md:45]: Description should be the **value** of the custom event data key/value pair - all of the other descriptions describe the value.

File: docs/Webhooks/Geo-triggering.md [hunk 2]
```diff
@@ -1,183 +1,207 @@
...
-  },
-  "eventMetaData": {
-      "eKey1": "eValue1"
-  }
-}
-```
+| Field                                 | Description                                                 |
+|---------------------------------------|-------------------------------------------------------------|
+| `zoneInfo.customData`                 | Custom data related to the zone.                            |
+| `zoneInfo.customData.[KEY]`         | The key of the zone custom data key/value pair.                    |
```
Comment (andrew-stirling) [inline docs/Webhooks/Geo-triggering.md:68]: Similar to above, should be value, rather than key.

**Comment by antong88** [comment]
What about dynamic webhook headers?

File: docs/Webhooks/Geo-triggering.md (hunk 1 above)
Comment (toroduque) [reply docs/Webhooks/Geo-triggering.md]: Updated!

File: docs/Webhooks/Geo-triggering.md (hunk 2 above)
Comment (toroduque) [reply docs/Webhooks/Geo-triggering.md]: Updated!

**Comment by toroduque** [comment]
> What about dynamic webhook headers?

@antong88 This goes in the Overview page, as setting the custom headers and body fields are not exclusive to the Geo-triggering event.

**Review by antong88 — APPROVED** [review]

**Pull request merged**